sdks/python/apache_beam/transforms/trigger.py

```diff
@@ -633,9 +685,14 @@ def to_runner_api(self, context):
 
 class TriggerContext(object):
 
-  def __init__(self, outer, window):
+  def __init__(self, outer, window, clock):
     self._outer = outer
     self._window = window
+    self._clock = clock
+
+  @property
```

Comment:
<!--new_thread; commit:5ddb81e7534625fa7f44e5ab7dbada5eb605bf3b; resolved:0-->
Let's not make this `@property` to make it clearer that this is actually retrieving the time on each call.